apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -397,7 +402,12 @@ export const pageRouter = router({
             message: "Failed to schedule page",
           })
         }
-        // TODO: add logic to add the job to the job queue
+        // add to the scheduled publish queue so it's processed by the worker at the scheduled time
```

Comment:
super nit (prob. bias) - i think no need for comment as function name is self-commenting